okio/src/commonMain/kotlin/okio/ByteString.kt

```diff
@@ -133,6 +133,14 @@ internal constructor(data: ByteArray) : Comparable<ByteString> {
    */
   fun rangeEquals(offset: Int, other: ByteArray, otherOffset: Int, byteCount: Int): Boolean
 
+  /**
+   * Copies bytes of this in `[offset..offset+byteCount]` to other in
+   * `[otherOffset..otherOffset+byteCount]`.
+   *
+   * @throws IndexOutOfBoundsException if either range is out of bounds.
+   */
+  fun copyInto(offset: Int = 0, target: ByteArray, targetOffset: Int = 0, byteCount: Int)
```

Comment:
Any reason not to make `byteCount` equal to `size` by default?

Reply:
I think it’s too difficult because ... which size? It’s uncertain whether source or target is larger

Reply:
Is your Wire use case "copy all bytes in `this.data` into `target`", or something different?

Reply:
it’s copying an entire ByteString into a specific range of a target ByteArray